src/openloco/config.cpp

```diff
@@ -138,6 +140,7 @@ namespace openloco::config
         node["loco_install_path"] = _new_config.loco_install_path;
         node["language"] = _new_config.language;
         node["breakdowns_disabled"] = _new_config.breakdowns_disabled;
+        node["companyAIDisabled"] = _new_config.companyAIDisabled;
```

Comment:
I know I did just tell you to change to camelCase but do we want the config file to be camelCase?

Reply:
It makes sense for the YAML config file to reflect the struct, right?

Reply:
I don't know, does yml have a default style?